python/export_util.py

```diff
@@ -235,3 +311,287 @@ def csv_query(
         data = csv_to_stream(data_list)
 
     return mgp.Record(file_path=file_path, data=data)
+
+
+def write_header(output: io.StringIO):
+    output.write('<?xml version="1.0" encoding="UTF-8"?>\n')
+    output.write(
+        '<graphml xmlns="http://graphml.graphdrawing.org/xmlns" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://graphml.graphdrawing.org/xmlns http://graphml.graphdrawing.org/xmlns/1.0/graphml.xsd">\n'  # noqa: E501
+    )
+
+
+def translate_types(variable: Any):
+    if isinstance(variable, str):
+        return "string"
+    if isinstance(variable, bool):
+        return "boolean"
+    if isinstance(variable, float):
+        return "float"
+    if isinstance(variable, int):
+        return "int"
+    raise Exception(
+        "Property values can only be primitive types or arrays of primitive types."  # noqa: E501
+    )
+
+
+def check_if_elements_same_type(variable: Any):
+    for element in variable:
```

Comment:
You need to check here that variable is list and is not empty, otherwise exception will be raised on line 339 when acessing elements

Reply:
before calling this function I check if I call it for a tuple, i'm gonna change the typings

Reply:
Hmm, but I think that is part of this function, because someone else can forget that check and still call this function, so I would push that check here